packages/sit-onyx/src/components/OnyxRadioButton/types.ts

```diff
@@ -1,13 +1,14 @@
 import type { DensityProp } from "../../composables/density";
 import type { CustomValidityProp } from "../../composables/useCustomValidity";
-import type { BaseSelectOption, SelectOptionValue } from "../../types";
+import type { AutofocusProp, BaseSelectOption, SelectOptionValue } from "../../types";
 
 export type OnyxRadioButtonProps<TValue extends SelectOptionValue = SelectOptionValue> = Omit<
   BaseSelectOption<TValue>,
   "hideLabel"
 > &
   DensityProp &
-  CustomValidityProp & {
+  CustomValidityProp &
+  AutofocusProp & {
```

Comment:
don't you also need to updatet the `RadioButtonOption` type in 
`packages/sit-onyx/src/components/OnyxRadioButtonGroup/types.ts`
like in `packages/sit-onyx/src/components/OnyxCheckboxGroup/types.ts`?

Reply:
Good catch, thanks :)